src/pages/components/toggle.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { NamespacesConsumer } from 'react-i18next';
+import { withI18next } from '@wapps/gatsby-plugin-i18next';
+
+import Layout from 'components/layout';
+import SEO from 'components/seo';
+import { Toggle } from 'components/ExampleComponents';
+import ComponentDescription from 'components/ComponentDescription';
+import sideNavData from 'config/sidenav/components';
+import NoteBox from 'components/NoteBox';
+import Section from 'components/Section';
+import ComponentExample from 'components/ComponentExample';
+import { Heading, Text, Paragraph } from 'components/ResponsiveComponents';
+
+let checked = false;
+
+const getAriaLabelText = (t: Function): string => {
+  return t(`toggle.state.${checked ? 'on' : 'off'}`, {
+    name: t('toggle.label'),
+  });
+};
+
+const handleClick = (id: string, t: Function): void => {
+  checked = !checked;
+  document.getElementById(id).setAttribute('aria-label', getAriaLabelText(t));
```

Comment:
Is there need to dynamically change the **aria-label** based on the state? Or is there some other aria- that should be used in these kind of situations?

Currently it is like: 
- Toggle checked = **false** -> _Oheisteksti on kytketty pois päältä_
- Toggle checked = **true** ->  _Oheisteksti on kytketty päälle_

Reply:
Tagging myself here and resolving this why like this or do we fix it to be handled with component props @J-Kallunki

Reply:
@ketsappi So this way of using `.setAttribute` should be removed and also create issue of removing it from elsewhere.

Component should get the changed aria-label by updating prop 'aria-label'.

Reply:
I have now committed changes where .setAttribute is no longer in use in the toggle.tsx.

Reply:
> @ketsappi So this way of using `.setAttribute` should be removed and also create issue of removing it from elsewhere.

I created the issue for that and it can be found here: #87